rewrite-maven/src/main/java/org/openrewrite/maven/ChangeParentPom.java

```diff
@@ -205,12 +205,15 @@ public Xml.Tag visitTag(Xml.Tag tag, ExecutionContext ctx) {
                                     changeParentTagVisitors.add(new ChangeTagValueVisitor<>(t.getChild("relativePath").get(), targetRelativePath));
                                 }
                                 else if (tag.getChildValue("relativePath").orElse(null) == null && targetRelativePath != null) {
+                                    Xml.Tag relativePathTag = null;
```

Comment:
```suggestion
                                    final Xml.Tag relativePathTag;
```

Reply:
Since we're assigning in both paths of the if, and using the result regardless I figured make it explicit that `null` is never used or expected.